chapters/3-measurement.tex

```diff
@@ -270,57 +301,40 @@ \section{Relate research design to a data map}
 and your research design.
 We assume you have a working familiarity
 with the research designs mentioned here.
-If needed, you can reference \textcolor{red}{Appendix XYZ},
+You should reference our \textbf{Research Design Appendix},
 where you will find more details
-and specific references for common impact evaluation methods.
+and specific references for common impact evaluation designs.
 
 \subsection{Defining research variables related to your research design}
 
 As DIME primarily works on impact evaluations,
 we focus our discussion here on research designs
 that compare a group that received
-some kind of \textbf{treatment}\index{Treatment}\sidenote{
+some kind of \textbf{treatment}\index{treatment}\sidenote{
 	\textbf{Treatment:} The general word for the evaluated intervention or event.
 	This includes things like being offered a training,
 	a cash transfer from a program,
 	or experiencing a natural disaster, among many others.}
-against a counterfactual control group\index{Counterfactual}.\sidenote{
+group against a \textbf{counterfactual}\index{counterfactual}.\sidenote{
 	\textbf{Counterfactual:} A statistical description of
 	what would have happened
 	to specific individuals in an alternative scenario,
 	for example, a different treatment assignment outcome.}
-
 The key assumption is that each
 person, facility, or village
 (or whatever the unit of treatment is)
 had two possible states: their outcome if they did receive the treatment
 and their outcome if they did not receive that treatment.
-The average impact of the treatment, or the ATE\sidenote{
-	The \textbf{average treatment effect (ATE)}
-	is the expected average change in outcome
+The average impact of the treatment,\sidenote{
+	\textbf{Average treatment effect (ATE):}
+	The expected average change in outcome
 	that untreated units would have experienced
-	had they been treated.},
+	had they been treated.}
 is defined as the difference
 between these two states averaged over all units.
 
-However, we can never observe the same unit
-in both the treated and untreated state simultaneously,
-so we cannot calculate these differences directly.
-Instead, the treatment group is compared to a control group
-that is statistically indistinguishable,
-which makes the average impact of the treatment
-mathematically equivalent to
-the difference in averages between the groups.
-Statistical similarity is often defined
-as \textbf{balance} between two or more groups.
-Since balance tests are commonly run for impact evaluations,
-DIME Analytics created a Stata command to
-standardize and automate the creation of nicely-formatted balance tables:
-\texttt{iebaltab}\sidenote{
-	\url{https://dimewiki.worldbank.org/iebaltab}}.
-
```

Comment:
Are we taking this out or moving it somewhere else? I think we should mention _statistically indistinguishable_ and/or _balance_ somewhere.

Reply:
Attempt in ef4be82362b9a6fb0381e99689fb6c74b0f04028

Reply:
Can I add iebaltab? I think that is the most well used command in our suites of commands

Reply:
done! 579d0d5c975db90e540cbe1ebae13375f904970d